### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by moutons** [wakatime/atom-wakatime#7]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/usr/local/share/atom/resources/app/src/atom.js:55:11)
<unknown> (/home/mouton/.atom/packages/wakatime/lib/wakatime.coffee:46:28)
```

**Comment by alanhamlett** [comment]
Fixed in v4.0.0 with ac9d597119aaee4acb55b7db68833f2f9d7e0fd0

Thanks!

**Comment by moutons** [comment]
:+1:
